users/service.go

```diff
@@ -198,15 +201,21 @@ func (svc usersService) Login(ctx context.Context, user User) (string, error) {
 	return svc.issue(ctx, dbUser.Email, authn.UserKey)
 }
 
-func (svc usersService) User(ctx context.Context, token string) (User, error) {
+func (svc usersService) User(ctx context.Context, token, id string) (User, error) {
```

Comment:
this actually doesnt get user for ID only for token?

Reply:
For `id` if it's a valid UUID. If empty or not valid it uses Email